botocore/regions.py

```diff
@@ -96,13 +96,19 @@ class EndpointResolver(BaseEndpointResolver):
 
     _UNSUPPORTED_DUALSTACK_PARTITIONS = ['aws-iso', 'aws-iso-b']
 
-    def __init__(self, endpoint_data):
+    def __init__(self, endpoint_data, uses_builtin_data=False):
         """
+        :type path: dict
         :param endpoint_data: A dict of partition data.
+
+        :type uses_builtin_data: boolean
+        :param uses_builtin_data: Whether the endpoint data originates in the
+            package's data directory.
         """
         if 'partitions' not in endpoint_data:
             raise ValueError('Missing "partitions" in endpoint data')
```

Comment:
maybe add `key` just to clarify that the its a dictionary key not just some string?

Reply:
I see where you're coming from. But this line hasn't been changed in seven years and I see more potential problems caused by me changing it now than potential problems that would solve.